docs/testrunner-toolkit/configuration/espresso.md

```diff
@@ -533,7 +533,7 @@ Removes all shared states from the testing device's CPU and memory at the comple
 #### `useTestOrchestrator`
 <p><small>| OPTIONAL | BOOLEAN |</small></p>
 
-If set, the instrumentation will start with [Test Orchestrator version 1.1.1](https://developer.android.com/training/testing/junit-runner#using-android-test-orchestrator) in use.
+If set, the instrumentation will start with [Test Orchestrator version 1.1.1](https://developer.android.com/training/testing/junit-runner#using-android-test-orchestrator) in use. This field is only for real devices test.
```

Comment:
```suggestion
If set, the instrumentation will start with [Test Orchestrator version 1.1.1](https://developer.android.com/training/testing/junit-runner#using-android-test-orchestrator) in use. This field is only works with real devices, not emulators.
```

Reply:
I improved the wording a little. Overall I'm not sure if this stands out enough though? I mean, at the very least we need this line for sure. But a more prominent note at line https://github.com/saucelabs/sauce-docs/pull/535/files#diff-f6be674891187d3d6794502646bbe94cb578588eaa8bc92e32dcb7313b99e9b5R534 or in the note below?

Reply:
Let me update that a bit and see